### DISCUSSION TITLE
Welcome to api Discussions!

### DISCUSSION THREAD

**Discussion by mattmilesi** [fattureincloud/api#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by valmoz** [comment]
Please, if you have questions regarding our APIs feel free to open a new Discussion! This way, other users can search for the same info you needed!

**Reply by valmoz** [reply]
We also suggest to check this discussion: #310
